runtime/compiler/control/J9Options.cpp

```diff
@@ -2650,6 +2650,7 @@ bool J9::Options::feLatePostProcess(void * base, TR::OptionSet * optionSet)
             }
 
          self()->setOption(TR_FullSpeedDebug);
+         compInfo->getPersistentInfo()->setLocalSyncCompiles(false);
```

Comment:
The code needs to be #ifdef protected since I expect this to compile only on platforms where JITServer is enabled.